src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      data-qa ="big" 
+      class="bigForm form">
+      <label for="bigBar">
+        <img
+         src="./images/Search.svg"
+         alt="icon"
+         class="icon"
+        >
+        <input
+         type="text"
+         name="bigBar"
+         placeholder="Try &quot;Los Angeles&quot;"
+         class="bigBar"
+         data-qa="keypress"
+         autocomplete="off"
+        >
```

Comment:
```suggestion
        <input
          type="text"
          name="bigBar"
          placeholder="Try &quot;Los Angeles&quot;"
          class="bigBar"
          data-qa="keypress"
          autocomplete="off"
        >
```

checklist p5
not 1 but 2 spacces